tests/fixtures/drafts/columns-with-comments.yaml

```
@@ -0,0 +1,4 @@
+models:
+  Profession:
+    title: string:400 comment:"Some title for the profession"
+    description: string:400 nullable comment:'Some description for the profession'
```

I thought something like

```suggestion
    title: string:400 "comment:Some title for the profession"
    description: string:400 nullable 'comment:Some description for the profession'
```

should have worked out of the box?

---

@spaceemotion why is that?

---

Based on https://blueprint.laravelshift.com/docs/model-data-types/:

> When specifying an attribute or modifier value which contains a space, you must wrap the value in double quotes ("). For example, enum:Ordered,Completed,"On Hold". Blueprint will unwrap the value during parsing.

I thought this was a general thing for all attributes.

---

I see. It was. Or at least the double quotes. But in your example, you also quoted the _attribute_ (i.e. `comment`)